ui/src/app/views/workspace-edit/component.spec.ts

```diff
@@ -91,6 +93,50 @@ describe('WorkspaceEditComponent', () => {
       ]}).compileComponents();
   }));
 
+  it('should show a conflict-specific error when creating a name conflict workspace',
+    fakeAsync(() => {
+    spyOn(TestBed.get(Router), 'navigate');
```

Comment:
This may warrant adding `.and.doNothing()`? Just to be a little more explicit about behavior?

Reply:
So, there doesn't seem to be a `.and.doNothing()` or even just a `doNothing()`. I'll add a comment here just in case.

Reply:
Oh, okay! My bad. I thought there was, but may have thought wrong!

Reply:
Turns out we don't need that spy for these test cases. They are needed for other cases, but not these error ones.